doc/release-notes/rl-0.1.1.md

```diff
@@ -8,6 +8,6 @@ Specifically, this release includes:
 - fixed `flox search --json`
 - ported multiple functions from bash MVP to rust rewrite
 - updated channel subscriptions to span multiple machines
-- the parsing of gh subcommand args has been fixed
+- the parsing of `gh` subcommand args has been fixed
```

Comment:
@robinbrantley @jurraca do we want to edit past release notes or just leave as is?

Reply:
@mkenigs I can revert that commit to the relnotes in my branch if you all decide you want to leave it as is.  I just saw it so I figured I'd submit it.

Reply:
> @robinbrantley @jurraca do we want to edit past release notes or just leave as is?

I'm fine with making the change, thanks for spotting it! :-)